cla_public/templates/privacy.html

```diff
@@ -54,82 +54,53 @@ <h2>{{ _('How is my privacy protected?') }}</h2>
 
   <p>
     {% trans %}To provide you with the best service that we can, Civil Legal Advice may share some information about you
-    with other organisations. But we will only do this if you have given us permission.{% endtrans %}
+    with other organisations. But we will only do this, if you have given us permission or when we have a lawful basis for doing so.{% endtrans %}
   </p>
 
   <p>
     {% trans %}In exceptional circumstances, we may share limited information about you without your consent. This would
-    only be to prevent fraud, help detect crime, if it is required by law or by a court order or If we are
+    only be to prevent fraud, to help detect crime, if it is required by law or by a court order or if we are
     very worried about your safety or the safety of others.{% endtrans %}
   </p>
 
-  <p>
-    {{ _('Read the full Civil Legal Advice Privacy Statement here.') }}
-  </p>
-
-
-  <h2{{ _('Civil Legal Advice Privacy Statement') }}></h2>
   <p>
     {% trans %}In order to provide a service to you, Civil Legal Advice (<abbr title="Civil Legal Advice">CLA</abbr>)
     may need to view, collect, record and hold (‘process’) personal data about you.{% endtrans %}
   </p>
   <p>
-    {% trans %}Personal data is information about you that can be used to identify you in some way
-    e.g. your name or telephone number. You can look at the
-    <a href="http://ico.org.uk/for_organisations/data_protection/the_guide/key_definitions#personal-data" rel="external">full legal definition</a>.{% endtrans %}
-  </p>
-  <p>
-    {% trans %}<abbr title="Civil Legal Advice">CLA</abbr> is committed to protecting your personal data and respecting your privacy.
-    We are legally bound by the Data Protection Act 1998 to ensure that we only use your personal
-    data for the reasons we tell you about. We must also hold it securely.{% endtrans %}
+    {% trans %}Personal data is information about you as an individual. It can be your name, address or telephone number. It can also include the information that you have provided in this form, such as your financial circumstances.{% endtrans %}
```

Comment:
Because you are using an opening and closing trans tag, you can split this line in two. Just like the ones above have been done.